biggerquery/user_commons/labels.py

```diff
@@ -0,0 +1,14 @@
+from biggerquery.interactive import interactive_component
```

Comment:
Why it is standalone file? Why you didnt add it to dataset_manager?

Reply:
Because there is no reason for making the dataset_manager even bigger. I think we should go in a more modular direction, instead of creating one huge class.

Reply:
👍